Classes/EddaConstants.cs

```diff
@@ -146,12 +155,13 @@ public static class Waveform {
         }
 
         public static class Spectrogram {
-            public const int AmplitudeScale = 16_000;
-            public const int Width = 100;
-            public const int MaxFreq = 16_000; // Hz
+            public const int MelBinCount = 100;
```

Comment:
I'm not sure if we should also make this available for users when the type is MelScale. 

While increasing it will slightly improve the resolution, it also has a large impact on the contents, as it's basically a logarithmic scale - most of the added pixels are spent on very low (below 100Hz), uninteresting frequencies.

Reply:
this is probably something we should hide from the user.

we can tune this value ourselves for quality/performance and it should be able to work well for any song.

Reply:
Values over 100 don't increase the quality significantly, but cause the logarithmic drift I've described above:

100:
![image](https://user-images.githubusercontent.com/12004018/225467306-77c16650-0ef3-4aca-8438-61f47abe837c.png)

150:
![image](https://user-images.githubusercontent.com/12004018/225467178-7fe7fee0-0ee9-460c-89a8-ffd3a2dfb135.png)

200:
![image](https://user-images.githubusercontent.com/12004018/225467399-11a7f799-377a-4e16-99d9-7255c8d95c7b.png)